lib/osf-components/addon/components/files/selected-list/template.hbs

```diff
@@ -7,7 +7,7 @@
             <AnimatedEach
                 @items={{@selectedFiles}}
                 @use={{this.transition}}
-                @duration=500
+                @duration='500'
```

Comment:
Got a question: for these type of parameters that expect integers/numbers, can we use `{{500}}`? `'500'` makes it look like a string.